impeller/entity/shaders/solid_fill.frag

```diff
@@ -5,11 +5,11 @@
 #include <impeller/types.glsl>
 
 uniform FragInfo {
-  vec4 color;
+  f16vec4 color;
```

Comment:
Doesn't this require `#extension GL_EXT_shader_16bit_storage : require` up top?

Reply:
It does not, I'm not actually using a f16vec4 on glsl, here is the compiled shader:

```
#version 100
precision mediump float;
precision highp int;

struct FragInfo
{
    vec4 color;
};

uniform FragInfo frag_info;

void main()
{
    gl_FragData[0] = frag_info.color;
}
```

Reply:
oh - but this is all in types.glsl too